### ISSUE TITLE
Visibility and display properties merged together

### ISSUE THREAD

**Issue by jeremylevy** [jakubpawlowicz/clean-css#674]
Hello,

Visibility and display properties seem to be merged together:

```js
var CleanCSS = require('clean-css');

new CleanCSS().minify('.b { display: none } .b { display: none; visibility: hidden; }').styles;

-> .b{visibility:hidden}
```

Thanks!

**Comment by jakubpawlowicz** [comment]
It's a bug indeed. Fix is coming!

**Comment by jakubpawlowicz** [comment]
Fixed in 3.4.4.
